reactDashboard/src/pages/Accessibility.jsx

```diff
@@ -5,14 +5,15 @@ import "bootstrap/dist/css/bootstrap.min.css";
 const Accessibility = () => {
   return (
     <div fluid="true" className="accessibility-page">
-      <div className="accessibility-intro">
+      <div className="accessibility-details">
+        <hr className="full-width-hr" />
         <h1>Accessibility</h1>
         <hr className="full-width-hr" />
         <p>
           We want everyone who visits the Covid-19 Dashboard to feel welcome,
-          and able to find information whatever their circumstances.
+          and able to find information whatever their circumstances. To do this
```

Comment:
This wasn't part of the job for dark mode. Stop adding unrelated changes. If you want to modify it, fine. Put it in a different PR

Reply:
This was dealing with Rob's comments about extra lines/busy interface - the lines have been left in and the layout was simplified and colours were simplified instead

Reply:
Fair enough